src/webserver.ts

```diff
@@ -22,7 +22,7 @@ Sentry.init({
   enabled: true, // might toggle this conditionally at some point
   dsn: process.env.SENTRY_DSN,
   debug: process.env.NODE_ENV !== "production" ? true : false,
-  release: "add-sentry", //todo automate this??
+  release: "p4nth3rb0t@" + process.env.npm_package_version,
```

Comment:
Do we need to preface the release with `p4nth3rb0t@` given it is namespaced in Sentry?

Reply:
Good find. I assumed this but it might very well be just the version:
```suggestion
  release: process.env.npm_package_version,
```